src/Native/Runtime/GCHelpers.h

```diff
@@ -0,0 +1,107 @@
+//
+// Copyright (c) Microsoft Corporation.  All rights reserved.
```

Comment:
I have just noticed that GCHelpers.cpp is `Unmanaged helpers exposed by the System.GC managed class.`. We should give this file a different name, maybe GCWriteBarriers.cpp ?